tools/mo/openvino/tools/mo/ops/split.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, graph: Graph, attrs: dict):
             'type': self.op,
 
             'infer': self.infer,
+            'reverse_infer': reverse_bypass_infer,
```

Comment:
please correct. now you set output shape to input. Please create dedicated function. `reverse_bypass_infer` is not suitable here

Reply:
Done. Left more detailed answer in the comment below